tests/unittests/api/helpers/test_files.py

```diff
@@ -125,6 +127,9 @@ def test_create_save_image_sizes(self):
             width_icon = 75
             image_sizes = create_save_image_sizes(image_url_test, image_sizes_type)
 
+            for image in image_sizes:  # So that file names below don't contain port spec.
+                image_sizes[image] = re.sub(r'\:(\d+)|None/', '/', image_sizes[image])
```

Comment:
And here. Are these the right places?

Reply:
As far as the standard is concerned, a URL-port string can contain _zero_ or more digits. This would not be stripped and stripping just `:` would not make much sense. Imho, we should use [`urllib.parse`](https://docs.python.org/3.0/library/urllib.parse.html) here to remove the port

Reply:
@maxlorenz I cannot think of a string which would break the above regex. Can you please provide an example that will? I'm open to using `urllib.parse`, if that's more readable and Pythonic.

Reply:
Well the only I can find would be the empty port example `http://test.com:/`. But urllib.parse seems more pythonic to me (the existing solution should work too, however). Maybe put the regex in a helper function for readability

Reply:
@maxlorenz I suspect such urls would never be generated. Also, I'm not able to think of a one-liner involving `urllib.parse`, so maybe wrapping in a helper function is the right way to go?

Reply:
@maxlorenz Anyways, are these the right places for stripping the ports off?

Reply:
@schedutron What does the function return. Can you paste the content here? I'm trying to wrap my head around as to why is there a need to strip ports.

Reply:
@iamareebjamal If the ports aren't stripped, the tests code tries looking for file paths with ports, and thus fails.

Reply:
Got it, use urlparse here as well

Reply:
```python
image_sizes = [urlparse(url).path for url in create_save_image_sizes(image_url_test, image_sizes_type)]